ContentLock/Client/src/localizations/fr.ts

```diff
@@ -10,20 +10,20 @@ export default {
         pagesCheckedOutTitle: 'Pages verrouillées',
     },
     contentLockFooterApp: {
-        lockedByYou: 'Cette page est verrouillée par vous',
+        lockedByYou: 'Vous avez verrouillé cette page',
         lockedByAnother: 'Cette page est verrouillée par {0}',
     },
     contentLockNotification: {
         lockedHeader: 'Contenu verrouillé',
-        lockedMessage: 'Le document a été verrouillé pour que vous puissiez le modifier.',
+        lockedMessage: 'Le document a été verrouillé afin que vous puissiez le modifier.',
         unlockedHeader: 'Contenu déverrouillé',
         unlockedMessage: 'Le document a été déverrouillé pour permettre à d’autres utilisateurs de le modifier.',
         bulkUnlockHeader: 'Contenu déverrouillé',
         bulkUnlockMessage: 'Le contenu sélectionné a été déverrouillé avec succès'
     },
     contentLockPermission: {
-        group: 'Content Lock', // TODO: Currently not used in Umbraco but added for future use
-        label: 'Unlocker',
+        group: 'Verrouillage de Contenu', // TODO: Currently not used in Umbraco but added for future use
+        label: 'Débloqueur',
         description: 'Permet au groupe d’utilisateurs de déverrouiller un document verrouillé par un autre utilisateur.',
```

Comment:
@mikecp I think for now I won't translate this directly, as its tied directly to the package name, so hence they have English values.